README.adoc

```diff
@@ -1,40 +1,99 @@
 = Jekyll Asciidoc Quickstart
 
-TODO
+= Congratulations!
 
-== Deploy site to GitHub Pages
+You've taken the first step down an easy path of blogging with Jekyll. Not only that, you've chosen AsciiDoc over Markdown which will give you fine control of the structure of your posts, and will make them display beautifully when you publish them using the default mobile-first stylesheets included in this repo.
 
-GitHub doesn’t (yet) whitelist the AsciiDoc plugin, so you can only run it on your own machine… or use a continuous integration platform like https://travis-ci.org/[Travis CI].
+= A Note About GitHub Pages
 
-You need two branches in your repository:
+GitHub Pages does not (yet) whitelist the jekyll-asciidoc plug-in, so you can not write `.adoc` posts and have them instantly publish like Markdown posts do. 
 
-* **master** for markup sources and configuration (may be named differently),
-* **gh-pages** for generated static content (will be created automatically for you).
+You can help change this by creating a support case through http://github.com/support and telling the GitHub team that you want the choice to write in AsciiDoc and have it handled the same as Markdown when pushed to your GitHub Page. Your voice counts: make it heard!
 
-The goal is to configure Travis CI job to listen for commits on the _master_ branch, automatically run jekyll build and push the generated content to the _gh-pages_ branch.
+== So what does that mean?
 
-. Install travis gem:
+Unlike some "fork and write" repos that exist, you need to initially configure the repo with a computer to publish using AsciiDoc. After initially configuring the repo, you can use Git command-line, or even a Git client on your tablet or smartphone to write, commit, and automatically publish blog posts.
```

Comment:
I like pointing this out, but I do think one of the first steps should be: for this repository.